src/components/DrawerSection.tsx

```diff
@@ -248,6 +253,7 @@ export const PermissionsDrawerSectionEntry = ({
                 width="120px"
                 flexShrink="0"
                 paddingTop="2px" // compensates for the extra height of tag components
+                fontWeight="500"
```

Comment:
Out of curiosity, why is this needed? What font does it default to?

Reply:
Defaults to 400 :)